gui.go

```diff
@@ -70,7 +74,8 @@ type Gui struct {
 
 // NewGui returns a new Gui object with a given output mode.
 func NewGui(mode OutputMode) (*Gui, error) {
-	if err := termbox.Init(); err != nil {
+	err := termbox.Init()
```

Comment:
Might not be necessary to split this up

Reply:
I split it because I want to use the err in L90

Reply:
Ah okay, resolving this then